### PR TITLE
Website #73, Update notifications & features #69

### PR DESCRIPTION
**Pull request by balthazzar**
https://github.com/webRunes/Chess-WRIO-Game/issues/73
https://github.com/webRunes/Chess-WRIO-Game/issues/69

### CODE REVIEW

File: wriologin.js [hunk 1]
```diff
@@ -0,0 +1,85 @@
+"use strict";
```
Comment (AlexeyAnshakov) [inline wriologin.js]: We have to auth using Login app. @michbil please advise

Comment (michbil) [inline wriologin.js]: use loginWithSessionId function to auth, example can be seen in Titter or Storage or webgold application. Skype me if any questions.

File: index.htm [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+    </script>
+    <script type="application/ld+json">
+        { "@context": "http://schema.org", "@type": "ItemList", "name": "My cover list for hub", "itemListElement": [ { "@type": "ItemList", "name": "Cover", "description": "Info about list", "image": "wrio.s3-website-us-east-1.amazonaws.com/Default-WRIO-Theme/img/no-photo-200x200.png", "url": "http://webrunes.com/cover.htm?cover" } ] }
+    </script>
+    <script src="//code.jquery.com/jquery-1.11.2.min.js"></script>
```
Comment (AlexeyAnshakov) [inline index.htm]: Удалить + 2 ссылки ниже

File: index.htm [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+    <link href="//maxcdn.bootstrapcdn.com/bootstrap/3.3.2/css/bootstrap.min.css" rel="stylesheet">
+    <link href="//wrio.s3-website-us-east-1.amazonaws.com/Default-WRIO-Theme/css/webrunes.css" rel="stylesheet">
+</head>
+
+<body>
+    <script type="text/javascript" src="//wrioos.com/start.js"></script>
+    <script src="//maxcdn.bootstrapcdn.com/bootstrap/3.3.2/js/bootstrap.min.js"></script>
```
Comment (AlexeyAnshakov) [inline index.htm]: Удалить

**Pull request merged**